open-metadata-resources/open-metadata-archives/open-metadata-types/src/main/java/org/odpi/openmetadata/opentypes/OpenMetadataTypesArchive.java

```diff
@@ -165,9 +159,63 @@ public void getOriginalTypes()
          * Calls for new and changed types go here
          */
         update0015LinkedMediaTypes();
+
+        add0042ProcessingStateClassification();
     }
 
 
+    /*
+     * -------------------------------------------------------------------------------------------------------
+     */
+
+
+    /**
+     * 0043  Add the Processing State Classification
+     */
+    private void add0042ProcessingStateClassification()
+    {
+        this.archiveBuilder.addClassificationDef(addProcessingStateClassification());
+    }
+
+    private ClassificationDef addProcessingStateClassification()
+    {
+
+        final String guid = "261fb0aa-b884-4ee8-87ea-a60510e9751d";
+        final String name = "ProcessingState";
+        final String description = "Stores last sync timestamp for critical elements.";
+        final String descriptionGUID = null;
+
+        final String linkedToEntity = "SoftwareCapability";
+
+        ClassificationDef classificationDef = archiveHelper.getClassificationDef(guid,
+                name,
+                null,
+                description,
+                descriptionGUID,
+                this.archiveBuilder.getEntityDef(linkedToEntity),
+                true);
+
+        /*
+         * Build the attributes
+         */
+        List<TypeDefAttribute> properties = new ArrayList<>();
+        TypeDefAttribute property;
+
+        final String attribute1Name = "syncDatesByKey";
+        final String attribute1Description = "The sync dates map of critical elements and sync status.";
```

Comment:
Similarly here, I would re-phrase to more generic attribute description:  
"Collection of synchronization dates identified by a key"